app-crypt/gnupg/gnupg-2.2.21.recipe

```diff
@@ -105,6 +104,8 @@ TEST_REQUIRES="
 	cmd:ssh
 	"
 
+PATCHES="gnupg-2.2.21.patchset"
```

Comment:
Please move this back to the line after the CHECKSUM (eg. into line 20)

Reply:
And please use the $portVersion variable instead of hardcoding the version number here.

Reply:
Requested changes made.